diff --git a/node_modules/@discordjs/voice/dist/index.js b/node_modules/@discordjs/voice/dist/index.js
index 7ddadee..7ca87cb 100644
--- a/node_modules/@discordjs/voice/dist/index.js
+++ b/node_modules/@discordjs/voice/dist/index.js
@@ -1359,6 +1359,7 @@ var VoiceConnection = class extends import_node_events7.EventEmitter {
   }
   addStatePacket(packet) {
     this.packets.state = packet;
+    this.configureNetworking();
     if (typeof packet.self_deaf !== "undefined")
       this.joinConfig.selfDeaf = packet.self_deaf;
     if (typeof packet.self_mute !== "undefined")
